script/fetch-recorded-data

```diff
@@ -189,10 +193,28 @@ async function main() {
       `,
       [caseRecords]
```

Comment:
Is case data likely to be empty too? Should we have the same emptiness check there?